cli/src/main/java/org/jboss/as/cli/impl/aesh/HelpSupport.java

```diff
@@ -168,4 +211,846 @@ public static String format(CommandContext ctx, BufferedReader reader) throws IO
     }
+
+    static final Map<Class<?>, String> VALUES = new HashMap<>();
+
+    static {
+        VALUES.put(BigDecimal.class, "a big decimal");
+        VALUES.put(Boolean.class, "true|false");
+        VALUES.put(BigInteger.class, "a big integer");
+        VALUES.put(byte[].class, "bytes array");
+        VALUES.put(Double.class, "a double");
+        VALUES.put(Expression.class, "an expression");
+        VALUES.put(Integer.class, "an integer");
+        VALUES.put(List.class, "a list");
+        VALUES.put(Long.class, "a long");
+        VALUES.put(Object.class, "an object");
+        VALUES.put(Property.class, "a property");
+        VALUES.put(String.class, "a string");
+        VALUES.put(ModelType.class, "a type");
+    }
+
+    private static class Expression {
+
+    }
+
+    public static Class<?> getClassFromType(ModelType mt) {
+        Class<?> clazz = String.class;
+        switch (mt) {
+            case BIG_DECIMAL: {
+                clazz = BigDecimal.class;
+                break;
+            }
+            case BOOLEAN: {
+                clazz = Boolean.class;
+                break;
+            }
+            case BIG_INTEGER: {
+                clazz = BigInteger.class;
+                break;
+            }
+            case BYTES: {
+                clazz = byte[].class;
+                break;
+            }
+            case DOUBLE: {
+                clazz = Double.class;
+                break;
+            }
+            case EXPRESSION: {
+                clazz = Expression.class;
+                break;
+            }
+            case INT: {
+                clazz = Integer.class;
+                break;
+            }
+            case LIST: {
+                clazz = List.class;
+                break;
+            }
+            case LONG: {
+                clazz = Long.class;
+                break;
+            }
+            case OBJECT: {
+                clazz = Object.class;
+                break;
+            }
+            case PROPERTY: {
+                clazz = Property.class;
+                break;
+            }
+            case STRING: {
+                clazz = String.class;
+                break;
+            }
+            case TYPE: {
+                clazz = ModelType.class;
+                break;
+            }
+            default: {
+                clazz = String.class;
+                break;
+            }
+        }
+        return clazz;
+    }
+
+    public static String printHelp(CommandContext ctx, ModelNode mn,
+            OperationRequestAddress address) {
+        try {
+            // Build a ProcessedCommand from the ModelNode
+            String commandName = mn.get("operation-name").asString();
+            String desc = mn.get(Util.DESCRIPTION).asString();
+            desc += Config.getLineSeparator() + Config.getLineSeparator()
+                    + "NB: to retrieve operation full description call the following operation: "
```

Comment:
"to retrieve the full operation description invoke" reads better

Reply:
Ok, thanks.